File: Models/Responses/Budgets/ResponseCostCenter.cs
namespace Models.Responses.Budgets;

/// <summary>
/// 코스트 센터 응답 모델
/// </summary>
public class ResponseCostCenter : ResponseCommonWriter
{
    /// <summary>
    /// 아이디 
    /// </summary>
    public Guid Id { get; init; }

    /// <summary>
    /// DbModelCostCenter 값 (유니크)
    /// </summary>
    public string Value { get; init; } = "";
}

File: BudgetBringerV2/src/models/responses/budgets/response-cost-center.ts
import type { ResponseCommonWriter } from '@/models/responses/response-common-writer'

/**
 * 코스트 센터 응답 모델
 */
export interface ResponseCostCenter extends ResponseCommonWriter {
  /**
   * 아이디
   */
  id: string;

  /**
   * DbModelCostCenter 값 (유니크)
   */
  value: string;
}
